Urban-Fun/Cells/TimelineCell.m

```diff
@@ -6,21 +6,48 @@
 //
 
 #import "TimelineCell.h"
+#import <MapKit/MapKit.h>
 
 @implementation TimelineCell
 
 - (void)awakeFromNib {
     [super awakeFromNib];
     // Initialization code
+    [self.contentView.layer setBorderColor:(__bridge CGColorRef _Nullable)([UIColor colorNamed:@"Black"])];
 }
 
 - (void)setSelected:(BOOL)selected animated:(BOOL)animated {
     [super setSelected:selected animated:animated];
 
     // Configure the view for the selected state
 }
+
+- getCLLocation:(PFGeoPoint *)location{
```

Comment:
Add the return type for the function and better naming
`- (CLLocation *)getCLLocationForGeoPoint(PFGeoPoint *)geoPoint { ...`